modules/client-configuration/pages/channels.adoc

```diff
@@ -33,6 +33,12 @@ This provides the {phub} channels, which you can subscribe your clients to.
 
 On Salt clients, you must apply the highstate after subscribing to the channel to be able to install packages.
```

Comment:
This sentence should be above the PackageHub section, this is true for all Salt clients.